bin/dasht

```diff
@@ -67,11 +72,29 @@
 # Written in 2016 by Suraj N. Kurapati <https://github.com/sunaku/dasht>
 # Distributed under the terms of the ISC license (refer to README file).
 
+: ${DASHT_USE_DASH_APP:=false}
+if $DASHT_USE_DASH_APP; then
+  : ${DASHT_DOCSETS_DIR:="/Users/smackesey/Library/Application Support/Dash"}
+else
+  : ${DASHT_DOCSETS_DIR:=${XDG_DATA_HOME:-$HOME/.local/share}/dasht/docsets}
+fi
+
 if test $# -gt 0; then
   pattern=$1
   shift
 fi
 
+# Ensure any docsets we are accessing from the Dash.app library have had their
+# documents extracted. Dash.app leaves the documents in the tarball by default.
+if $DASHT_USE_DASH_APP; then
+  dasht-docsets "$@" | while read docset; do
+    root=("$DASHT_DOCSETS_DIR/"*/*"/${docset}.docset")
```

Comment:
Shouldn't need `(` array `)` syntax.

Reply:
Regarding [your comment](https://github.com/sunaku/dasht/pull/53#issuecomment-727646478):

> those particular globs won't expand in my bash v3.2.57 without the array syntax. Something about putting an * next to a quote, I think.

Could you please try surrounding the stars with slashes by moving them out of the strings?  Like this:

```sh
root="$DASHT_DOCSETS_DIR"/*/*/"${docset}.docset"
```

Reply:
Also, if BASH is requiring you to add `(` array `)` syntax because the glob can potentially match more than one file, then it would be better to convert this expression into a nested loop:

```sh
for root in "$DASHT_DOCSETS_DIR"/*/*/"$docset".docset; do
  # ...
done
```